templates/alfresco/abstract.html

```diff
@@ -1,141 +1,263 @@
-    <td>Relative Vegetation Change</td>
-    <td>These data are derived from the vegetation class data, the principal phenomena being modeled by ALFRESCO. They are computed as the average number of year-to-year transitions of any vegetation class for a pixel, across all reps for all years of the summary year span. Pixels are 1km x 1km.</td>
-  </tr>
-  <tr>
-    <td>veg_type</td>
-    <td>vt</td>
-    <td>Vegetation Type</td>
-    <td>These data are derived from the vegetation class data, the principal phenomena being modeled by ALFRESCO. They are computed as the average percentage of each vegetation type for a pixel, across all reps for all years of the summary year span. Pixels are 1km x 1km.</td>
-  </tr>
-</tbody>
+  <thead>
+    <tr>
+      <th>Endpoint Name</th>
+      <th>Data Layer</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>flammability</td>
+      <td>Flammability</td>
+      <td>
+        These data are derived from the fire scar outputs and are computed as
+        the average number of times a pixel burned across all reps for all years
+        of the summary year span. Pixels are 1km x 1km.
+      </td>
+    </tr>
+    <tr>
+      <td>veg_type</td>
+      <td>Vegetation Type</td>
+      <td>
+        These data are derived from the vegetation class data, the principal
+        phenomena being modeled by ALFRESCO. They are computed as the average
+        percentage of each vegetation type for a pixel, across all reps for all
+        years of the summary year span. Pixels are 1km x 1km.
+      </td>
+    </tr>
+  </tbody>
 </table>
 
 <h4>Service endpoints</h4>
 <p>Here are the existing endpoints for the above datasets.</p>
 <ul>
-    <li><a href="/alfresco/point">Point query</a></li>
-    <li><a href="/alfresco/local">Point-based aggregation of local area (HUC-12)</a></li>
-    <li><a href="/alfresco/area">Aggregate by AOI polygon ID</a></li>
+  <li><a href="/alfresco/point">Point query</a></li>
+  <li>
+    <a href="/alfresco/local">Point-based aggregation of local area (HUC-12)</a>
+  </li>
+  <li><a href="/alfresco/area">Aggregate by AOI polygon ID</a></li>
 </ul>
 
-<h4>Results from queries to the above endpoints will look like this:</h4>
+<h3>Flammability</h3>
+
+<p>Results from flammability endpoints will look like this:</p>
 
 <pre>
-{
-  "1950-1959": {
-    "CRU-TS40": {
-      "CRU_historical": {
-        "rf": 0.0009
+  {
+    "1950-1979": {
+      "MODEL-SPINUP": {
+        "historical": 0.0065
       }
+    },
+    ...
+    "2010-2039": {
+      "5modelAvg": {
+        "rcp45": 0.0014,
+        "rcp60": 0.0007,
+        "rcp85": 0.0007
+      },
+      ...
+    },
+    ...
+  }
+</pre>
+
+<p>The above output is structured as such:</p>
+
+<pre>
+{
+  &ltera>: {
+    &ltmodel>: {
+      &ltscenario>: &ltvalue>
     }
   }
-...
-  "2010-2019": {
-    "5modelAvg": {
+  ...
+}
+</pre>
+
+<h4>Dimensions:</h4>
+<ul>
+  <li>
+    <strong><code>era</code></strong
+    >: Sumary era:
```

Comment:
```suggestion
    >: Summary era:
```

Reply:
Fixed in 0496ce0a4c8459239911af85d431d4e3f0e10567.